web/pdf_page_view.js

```diff
@@ -559,7 +559,11 @@ class PDFPageView {
 
     const viewport = this.viewport;
     const canvas = document.createElement("canvas");
-    canvas.id = this.renderingId;
```

Comment:
I vaguely remember that this is/was used for cleaning up canvases if they are out of view, but most likely this has changed over the years. Could you check if that still works as expected with this patch? If so, I also think this is now unused; if it's easy to find, perhaps the Git logs will tell us why this was ever introduced?

Reply:
Are you perhaps thinking of https://github.com/mozilla/pdf.js/blob/4729fdc0e930154c9e4c4688b6239ed4708f0e5e/web/base_viewer.js#L84-L118
In that case that shouldn't be a problem since it's dealing with `PDFPageView` instances.
I'll try to check the blame as well, but given how the code has been refactored over the years that may be a bit tricky...

Reply:
OK, I've tried going back approx. 10 years in the history and I'm not able to easily spot why these DOM elements need an `id`. Obviously I could be missing something, but it really seems unused.

Reply:
Alright, then it seems like it's indeed unused. Thank you for checking!